framework/src/application/genesis_block.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+// TODO: Remove this file completely after #5234
+
+import { Block } from '@liskhq/lisk-chain';
```

Comment:
Why we needed this at the moment? It will be removed completely with new genesis bock. Should not we just use `any` or `object` somewhere for now.

Reply:
This function was there already to convert genesis block to actual instance.
but with the new genesis block, this whole file should be removed, not only the type